scripts/ai2_internal/run_with_beaker.py

```diff
@@ -152,7 +153,7 @@ def resume_command(experiment_id):
         experiment_id = subprocess.check_output(experiment_command, universal_newlines=True).strip()
         print(
             f"Experiment {experiment_id} submitted. "
-            f"See progress at https://allenai.beaker.org/ex/{experiment_id}"
+            f"See progress at https://beaker.org/ex/{experiment_id}"
```

Comment:
Switching since allenai.beaker.org is being migrated to beaker.org as we speak. This hardcoded reference is probably best avoided though?

Reply:
This is a very good switch ;-)

I don't understand your question.

Reply:
Not particularly important: `beaker configure` can point to multiple URLs but since allenai.beaker.org is going away, seems fine to emit the hardcoded URL...

Reply:
Thanks for catching this. And yes, if beaker.org is the only one in practice, let's just stick with the hardcoded URL.